src/Parser.js

```diff
@@ -74,849 +53,865 @@ function ParseFuncOrArgument(result, isFunction, token) {
-            n -= 1;
-        }
     }
-};
 
-/**
- * Parse a single symbol out of the string. Here, we handle both the functions
- * we have defined, as well as the single character symbols
- *
- * @return {?ParseFuncOrArgument}
- */
-Parser.prototype.parseSymbol = function() {
-    const nucleus = this.nextToken;
+    /**
+     * Parse a single symbol out of the string. Here, we handle both the functions
+     * we have defined, as well as the single character symbols
+     *
+     * @return {?ParseFuncOrArgument}
+     */
+    parseSymbol() {
+        const nucleus = this.nextToken;
 
-    if (functions[nucleus.text]) {
-        this.consume();
-        // If there exists a function with this name, we return the function and
-        // say that it is a function.
-        return new ParseFuncOrArgument(
-            nucleus.text,
-            true, nucleus);
-    } else if (symbols[this.mode][nucleus.text]) {
-        this.consume();
-        // Otherwise if this is a no-argument function, find the type it
-        // corresponds to in the symbols map
-        return new ParseFuncOrArgument(
-            new ParseNode(symbols[this.mode][nucleus.text].group,
-                          nucleus.text, this.mode, nucleus),
-            false, nucleus);
-    } else if (this.mode === "text" && cjkRegex.test(nucleus.text)) {
-        this.consume();
-        return new ParseFuncOrArgument(
-            new ParseNode("textord", nucleus.text, this.mode, nucleus),
-            false, nucleus);
-    } else if (nucleus.text === "$") {
-        return new ParseFuncOrArgument(
-            nucleus.text,
-            false, nucleus);
-    } else {
-        return null;
+        if (functions[nucleus.text]) {
+            this.consume();
+            // If there exists a function with this name, we return the function and
+            // say that it is a function.
+            return new ParseFuncOrArgument(
+                nucleus.text,
+                true, nucleus);
+        } else if (symbols[this.mode][nucleus.text]) {
+            this.consume();
+            // Otherwise if this is a no-argument function, find the type it
+            // corresponds to in the symbols map
+            return new ParseFuncOrArgument(
+                new ParseNode(symbols[this.mode][nucleus.text].group,
+                            nucleus.text, this.mode, nucleus),
+                false, nucleus);
+        } else if (this.mode === "text" && cjkRegex.test(nucleus.text)) {
+            this.consume();
+            return new ParseFuncOrArgument(
+                new ParseNode("textord", nucleus.text, this.mode, nucleus),
+                false, nucleus);
+        } else if (nucleus.text === "$") {
+            return new ParseFuncOrArgument(
+                nucleus.text,
+                false, nucleus);
+        } else {
+            return null;
+        }
     }
-};
+}
 
 Parser.prototype.ParseNode = ParseNode;
```

Comment:
Within the class you could have a line `ParseNode = ParseNode`.  This is similar to `static ParseNode = ParseNode` but adds the value to the prototype instead of the class itself.